Dungeon.java

```diff
@@ -0,0 +1,267 @@
+					sb.append("|");
+			}
+			sb.append("\n");
+			if (row != myMaze.length - 1)
+				sb.append(lineMaker("*", "-", myMaze[0].length));
+		}
+		sb.append(lineMaker("*", "*", myMaze[0].length));
+		
+		return sb.toString();
+	}*/
+	
+	/**
+	 * Returns a String representation of this instance.
+	 * 
+	 * @return a String representation of this instance.
+	 */
+	@Override
+	public String toString() {
+		// * * * * * * * * * * *
+		// * E | ? | ? | ? | ? *
+		// * - * - * - * - * - *
+		// * ? | ? | ? | ? | ? *
+		// * - * - * - * - * - *
+		// * ? | ? | ? | ? | ? *
+		// * - * - * - * - * - *
+		// * ? | ? | ? | ? | ? *
+		// * - * - * - * - * - *
+		// * ? | ? | ? | ? | ? *
+		// * * * * * * * * * * *
+		StringBuilder sb = new StringBuilder();
+		sb.append(lineMaker("*", myMaze[0].length));
+		for (int row = 0; row < myMaze.length; row++) {
+			sb.append(lineMaker(row));
+			if (row != myMaze.length - 1)
+				sb.append(lineMaker("-", myMaze[0].length));
+		}
+		sb.append(lineMaker("*", myMaze[0].length));
+		
+		return sb.toString();
+	}
+	
+	/**
+     * Creates and return a wall whose length and design is dependent on
+     * passed parameters.
+     *
+     * @param theSegment is the connecting link between *.
+     * @param theLength is the length of the wall.
+     *
+     * @return a String representation of the wall.
+     */
+	private String lineMaker(final String theSegment, final int theLength) {
+		StringBuilder sb = new StringBuilder();
+		
+		sb.append("*");
+		for (int i = 0; i < theLength; i++) {
+			sb.append(" ");
+			sb.append(theSegment);
+			sb.append(" ");
+			sb.append("*");
+		}
+		sb.append("\n");
+		return sb.toString();
+	}
+	
+	/* OLDER MORE CAPABLE LINEMAKER
+	 * You could specify each segment instead of one being * by default
+	 * 
+	private String lineMaker(final String theSegment, 
+							final String theSegment2, final int theLength) {
+		StringBuilder sb = new StringBuilder();
+		
+		sb.append(theSegment);
+		for (int i = 0; i < theLength; i++) {
+			sb.append(" ");
+			sb.append(theSegment2);
+			sb.append(" ");
+			sb.append(theSegment);
+		}
+		sb.append("\n");
+		return sb.toString();
+	}*/
+	
+	/**
+     * Creates and return a row of this instance's myMaze.
+     *
+     * @param theRow is a row number of this instance's myMaze.
+     *
+     * @return a String representation of the row.
+     */
+	private String lineMaker(final int theRow) {
+		StringBuilder sb = new StringBuilder();
+		sb.append("*");
+		for (int col = 0; col < myMaze[theRow].length; col++) {
+			sb.append(" ");
+			sb.append(myMaze[theRow][col]);
+			sb.append(" ");
+			if (col == myMaze[theRow].length - 1)
+				sb.append("*");
+			else
+				sb.append("|");
+		}
+		sb.append("\n");
+		return sb.toString();
+	}
+	
+	public void moveHero() {
+		/*
+		 * PLACEHOLDER
+		 * - Will need to update row 4 of myLocales to keep track of how the
+		 * hero is moving
+		 * 
+		 * - idea: accept a number and use a switch to alter the coordinates

```

Comment:
I like the idea of using a switch here for the movement- will be much cleaner than if-else statements

Reply:
I'm currently thinking of utilizing 2, 4, 6, 8 for the cardinal directions (based off of a normal numpad).

Reply:
Or WASD, whatever we end up deciding.